README.md

```diff
@@ -50,28 +47,39 @@ all:
 
 **Create a network**
 
-* `casper-node` & `casper-node-launcher` binaries (release) must be built before running `create-kube-network` 
+* `casper-node` & `casper-node-launcher` binaries (release) must be built before running `create-kube-network`
+
+```bash
+pushd casper-node-launcher || >&2 echo "casper-node-launcher dir expected"
+cargo build --release
+popd
+
+pushd casper-node || >&2 echo "casper-node dir expected"
+cargo build --release --package casper-node
```

Comment:
I should add build the client here too. The process fails without a built client. 
TODO